[Bug]: grade script clobbers manifestPlaceholder settings

### What happened?

The [build-extras-onesignal.gradle](https://github.com/OneSignal/OneSignal-Cordova-SDK/blob/95b735cd52070f7a8c8b4d7b8b18e8b916f23cb9/build-extras-onesignal.gradle#L17) script is replacing entries in `manifestPlaceHolders` of `build.gradle`.

This means that if you have a entry in `manifestPlaceholders` called `AUTH_URL_SCHEME` you will get a manifestMerger error:
`Manifest merger failed : Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <AUTH_URL_SCHEME> is provided.`

### Steps to reproduce?

```Markdown
1. Use the repo https://github.com/dtarnawsky/onesignal-807.git
2. npx ionic build && npx cap sync
3. npx cap run android

After build you get the error: 
Error: Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <AUTH_URL_SCHEME> is provided.
```

Note: If you remove onesignal-cordova-plugin then the application will run.

### What did you expect to happen?

The gradle script should not alter any entries in manifestPlaceholders of build.gradle

### OneSignal Cordova SDK version

onesignal-cordova-plugin 5.0.1

### Which platform(s) are affected?

- [ ] iOS
- [X] Android

### Relevant log output

```Shell
> Task :app:processDebugMainManifest FAILED
        /.../android/app/src/main/AndroidManifest.xml Error:
        Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for
        <AUTH_URL_SCHEME> is provided.
        
        See https://developer.android.com/r/studio-ui/build/manifest-merger for more information about the manifest
        merger.
        
        
        FAILURE: Build failed with an exception.
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

<!-- probot = {"onesignal-probot":{"response_time_in_business_days":5}} -->

--- a/build-extras-onesignal.gradle
+++ b/build-extras-onesignal.gradle
@@ -1,12 +1,5 @@
 import java.nio.file.Files
 
-android.defaultConfig {
-  manifestPlaceholders = [
-    onesignal_app_id: '', // Use from js code
-    onesignal_google_project_number: 'REMOTE'
-  ]
-}
-
 // Required for Android Support Library 26.0.0+ and Google Play services 11.+
 repositories {
   maven { url 'https://maven.google.com' }

